app/views/hyrax/dashboard/collections/new.html.erb

```diff
@@ -1,10 +1,14 @@
-<% provide :page_title, construct_page_title( t('.header', type_title: @collection.collection_type.title) ) %>
+<% provide :page_title, construct_page_title( t('.header', type_title:  @collection.collection_type.title) ) %>
```

Comment:
I'm not sure I understand the need for changes to this file for issue #2069.  Can you provide info on why these are needed?

Reply:
Okay, reverted.

Reply:
Reverted.